### DISCUSSION TITLE
Welcome to Ginger By Amdocs Discussions!

### DISCUSSION THREAD

**Discussion by MeniKadosh1** [Ginger-Automation/Ginger#1775] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by anil4684** [comment]
Hi Team,

Could please clarify on the below .

1. How can we do the document verification using Ginger ( Like verification of text in PDF) ?
2. please share the guideline document to create Custom plugin .

Thanks,
Anil

**Reply by JinendraGhodke** [reply]
Hi Anil,
1. PDF Text verification is currently not supported. 
2. Please find below the document on how to create a new plugin.
[Ginger Plugins KT Dev 2020.pptx](https://github.com/Ginger-Automation/Ginger/files/7023697/Ginger.Plugins.KT.Dev.2020.pptx)

Please let us know if you have any other queries
